packages/common/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -943,8 +943,7 @@ export class AudiusAPIClient {
 
     let headers = {}
     if (encodedCurrentUserId && getUnlisted) {
-      const { data, signature } =
-        await this.audiusBackendInstance.signDiscoveryNodeRequest()
```

Comment:
interesting, just wondering context

Reply:
`signDiscoveryNodeRequest` is specifically for doing oauth. `signData` is used everywhere else